.maintain/playbooks/run-integration-tests.yml

```diff
@@ -0,0 +1,202 @@
+---
+- hosts: "{{ target }}"
+  vars:
+    user: service
+    group: service
+    work_dir: /srv/composable-sandbox
+  tasks:
+
+  - name: Create a group
+    group:
+      name: "{{ user }}"
+      state: present
+    become: yes
+
+  - name: Create a user
+    user:
+      name: "{{ user }}"
+      state: present
+      shell: /bin/bash
+      home: "/home/{{ user }}"
+      group: "{{ group }}"
+    become: yes
+
+  - name: Create a work directory
+    ansible.builtin.file:
+      path: "{{ work_dir }}"
+      state: directory
+      mode: "0755"
+      owner: "{{ user }}"
+      group: "{{ group }}"
+    become: yes
+
+  - name: Download rustup-init.sh
+    get_url:
+      url: https://sh.rustup.rs
+      dest: "/home/{{ user }}/rustup-init.sh"
+      mode: '0440'
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Install Rust toolchain
+    ansible.builtin.shell: |
+      cat $HOME/rustup-init.sh | sh -s -- -y
```

Comment:
can this be call to some SH script or ansible shared step kind of? so we avoid duplicating it a lot?

Reply:
This isn't relevant already